### PR TITLE
Update `pathology_nuclei_classification` deterministic results

### PR DESCRIPTION
**Pull request by KumoLiu**
### Description
Update `pathology_nuclei_classification` deterministic results

- [x] update performance in README and metadata.json
- [x] update train.json (add lr scheduler)
- [x] update version in metadata.json
- [x] update model weights in large_files.yml
- [x] update graphs

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [x] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by yiheng-wang-nv — APPROVED** [review]
LGTM
